packages/gatsby-image/src/index.js

```diff
@@ -239,24 +241,51 @@ const listenToIntersections = (el, cb) => {
 }
 
 const noscriptImg = props => {
-  // Check if prop exists before adding each attribute to the string output below to prevent
-  // HTML validation issues caused by empty values like width="" and height=""
-  const src = props.src ? `src="${props.src}" ` : `src="" ` // required attribute
-  const sizes = props.sizes ? `sizes="${props.sizes}" ` : ``
-  const srcSet = props.srcSet ? `srcset="${props.srcSet}" ` : ``
-  const title = props.title ? `title="${props.title}" ` : ``
-  const alt = props.alt ? `alt="${props.alt}" ` : `alt="" ` // required attribute
-  const width = props.width ? `width="${props.width}" ` : ``
-  const height = props.height ? `height="${props.height}" ` : ``
-  const crossOrigin = props.crossOrigin
-    ? `crossorigin="${props.crossOrigin}" `
-    : ``
-  const loading = props.loading ? `loading="${props.loading}" ` : ``
-  const draggable = props.draggable ? `draggable="${props.draggable}" ` : ``
+  const {
+    src = ``,
+    alt = ``,
+    sizes,
+    srcSet,
+    title,
+    width,
+    height,
+    crossOrigin,
+    loading,
+    draggable,
+  } = props
```

Comment:
If leveraging the (almost) same destructure from `Img` isn't possible,  the two components should share what they can through a prop filter method, and then only handle the extra prop extraction that's unique. Note that this `noscript` variant appears to have fallen out of sync a little, and some props like `ariaHidden`(`aria-hidden`) may not spread onto `img` correctly? Spreading what is possible is probably a good idea though.